composer.json

```diff
@@ -17,9 +17,9 @@
         }
     ],
     "require": {
-        "doctrine/mongodb-odm-bundle": "~3.0",
-        "doctrine/mongodb-odm": "~1.0",
-        "sonata-project/admin-bundle": "~2.3"
+        "doctrine/mongodb-odm-bundle": "~3.0@dev",
```

Comment:
In that case, there should be some releases of this bundle or some develop branch since it's not great also in projects depend on master of this bundle.